internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -10,42 +10,81 @@ import (
+// LBFargateManifest holds the configuration to build a container image with an exposed port that receives
+// requests through a load balancer with AWS Fargate as the compute engine.
+type LBFargateManifest struct {
+	AppManifest     `yaml:",inline"`
+	Image           ImageWithPort `yaml:",flow"`
+	LBFargateConfig `yaml:",inline"`
+	Environments    map[string]LBFargateConfig `yaml:",flow"` // Fields to override per environment.
 }
 
-type LBFargateImage struct {
+// ImageWithPort represents a container image with an exposed port.
+type ImageWithPort struct {
 	AppImage `yaml:",inline"`
-	Port     int `yaml:"port"` // Port exposed in the container.
+	Port     int `yaml:"port"`
 }
 
-// NewLoadBalancedFargateManifest creates a new public load balanced Fargate service with minimal compute settings.
-func NewLoadBalancedFargateManifest(appName string, dockerfile string) *LoadBalancedFargateManifest {
-	return &LoadBalancedFargateManifest{
-		AppManifest: AppManifest{Name: appName, Type: LoadBalancedWebApplication},
-		Image: LBFargateImage{
+// LBFargateConfig represents a load balanced web application with AWS Fargate as compute.
+type LBFargateConfig struct {
+	RoutingRule      `yaml:"http,flow"`
+	ContainersConfig `yaml:",inline"`
+	Public           bool               `yaml:"public"`
+	Scaling          *AutoScalingConfig `yaml:",flow"`
+}
+
+// ContainersConfig represents the resource boundaries and environment variables for the containers in the service.
+type ContainersConfig struct {
+	CPU       int               `yaml:"cpu"`
+	Memory    int               `yaml:"memory"`
+	Count     int               `yaml:"count"`
+	Variables map[string]string `yaml:"variables"`
+	Secrets   map[string]string `yaml:"secrets"`
+}
+
+// RoutingRule holds the path to route requests to the service.
+type RoutingRule struct {
+	Path string `yaml:"path"`
+}
+
+// AutoScalingConfig is the configuration to scale the service with target tracking scaling policies.
+type AutoScalingConfig struct {
+	MinCount int `yaml:"minCount"`
+	MaxCount int `yaml:"maxCount"`
+
+	TargetCPU    float64 `yaml:"targetCPU"`
+	TargetMemory float64 `yaml:"targetMemory"`
+}
+
+// NewLoadBalancedFargateManifest creates a new public load balanced web service with an exposed port of 80, receives
+// all the requests from the load balancer and has a single task with minimal CPU and Memory thresholds.
+func NewLoadBalancedFargateManifest(appName string, dockerfile string) *LBFargateManifest {
+	return &LBFargateManifest{
+		AppManifest: AppManifest{
+			Name: appName,
+			Type: LoadBalancedWebApplication,
+		},
+		Image: ImageWithPort{
 			AppImage: AppImage{
 				Build: dockerfile,
 			},
-			Port: 8080,
+			Port: 80,
+		},
+		LBFargateConfig: LBFargateConfig{
+			RoutingRule: RoutingRule{
+				Path: "*",
+			},
+			ContainersConfig: ContainersConfig{
+				CPU:    256,
+				Memory: 512,
+				Count:  1,
+			},
```

Comment:
Should we include default Scaling configuration values?

Reply:
I wasn't sure what's your thought on it? enabling autoscaling by default seemed overkill to me but I don't know :P

Reply:
We could set and expose values that don't actually scale (min and max of 1) to make it obvious where you'd crank up those values if you wanted to assuming there's no inherent cost in doing so.

Reply:
I'll keep it in the comments for now, we'll most likely tweak things in the upcoming weeks anyway :)